.eleventy.js

```diff
@@ -256,6 +257,11 @@ module.exports = function (config) {
     config.addTransform('minifyHtml', minifyHtml);
   }
 
+  config.on('afterBuild', () => {
+    const {hashList} = require('./src/site/_data/helpers');
+    fs.writeFileSync('script-hash-list.json', JSON.stringify([...hashList]));
```

Comment:
Could we generate `firebase.json` in this function to eliminate the need for a temporary file and changes to `npm run dev`?